store/sqlstore/upgrade.go

```diff
@@ -1683,3 +1685,55 @@ func hasMissingMigrationsVersion600(sqlStore *SqlStore) bool {
 
 	return false
 }
+
+func upgradeDatabaseToVersion610(sqlStore *SqlStore) {
+	if hasMissingMigrationsVersion610(sqlStore) {
+		sqlStore.AlterColumnTypeIfExists("Sessions", "Roles", "text", "varchar(256)")
+		sqlStore.AlterColumnTypeIfExists("ChannelMembers", "Roles", "text", "varchar(256)")
+		sqlStore.AlterColumnTypeIfExists("TeamMembers", "Roles", "text", "varchar(256)")
+	}
+
+	// if shouldPerformUpgrade(sqlStore, Version5380, Version600) {
+	// 	saveSchemaVersion(sqlStore, Version610)
+	// }
+}
+
+func hasMissingMigrationsVersion610(sqlStore *SqlStore) bool {
+	textFn := func(table, column string) bool {
+		info, err := sqlStore.GetColumnInfo(table, column)
+		if err != nil {
+			mlog.Error("Error getting column info for migration check",
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.